<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widgetry::DataSelectorWidget</class>
 <widget class="QWidget" name="Widgetry::DataSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="searchBarLayout">
     <item>
      <widget class="QLineEdit" name="searchInput">
       <property name="placeholderText">
        <string>Search...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="icon">
        <iconset resource="../resources/widgetry_resources.qrc">
         <normaloff>:/icons/action_refresh.png</normaloff>:/icons/action_refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionGoToSearchBar">
   <property name="text">
    <string>Go to search bar</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionTriggerSearch">
   <property name="text">
    <string>Trigger Search</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Space</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>searchInput</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/widgetry_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchInput</sender>
   <signal>returnPressed()</signal>
   <receiver>refreshButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGoToSearchBar</sender>
   <signal>triggered()</signal>
   <receiver>searchInput</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reject()</slot>
 </slots>
</ui>
